src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs

```diff
@@ -1,20 +1,64 @@
-﻿using System;
+﻿// <auto-generated />
+using System;
+using Company.WebApplication1.Data;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure;
-using Microsoft.EntityFrameworkCore.Metadata;
 using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
 
 namespace Company.WebApplication1.Data.Migrations
 {
     [DbContext(typeof(ApplicationDbContext))]
-    [Migration("00000000000000_CreateIdentitySchema")]
+    [Migration("20180420230759_CreateIdentitySchema")]
```

Comment:
So I guess we hand fix this to 000000 for now?

Reply:
I guess I can just try to update the batch to do this replacement since its one line, replace `[Migration(@` => `[Migration("00000000000000_CreateIdentitySchema")]`

Reply:
Good idea--easy enough to pattern match